js/README.md

```diff
@@ -32,626 +32,66 @@
 
 # JavaScript Guide
 
-Clean, consistent and understandable CSS is paramount to a successful project. It sets the tone, both visually and architecturally, for the entire front-end. These are our collected (and evolving) best practices for writing styles with some _style_. 😎
+*A mostly reasonable approach to JavaScript, inspired by the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript).*
 
-- [Organization](#organization)
-  - [Directory Structure](#directory-structure)
-  - [Base Styles](#base-styles)
-  - [Components](#components)
-  - [Utilities](#utilities)
-- [Conventions](#conventions)
-  - [General Rules](#general-rules)
-  - [Property Sorting](#property-sorting)
-  - [Selectors](#selectors)
-  - [Class Naming](#class-naming)
-  - [Comments](#comments)
-- [Architecture](#architecture)
-  - [Dryness](#dryness)
-  - [Encapsulation](#encapsulation)
-  - [Composition](#composition)
-  - [Compatibility](#compatibility)
-- [Tools](#tools)
-  - [my](#blah)
-  - [Processors](#processors)
-  - [Formatters](#formatters)
-  - [Validators](#validators)
-  - [Analyzers](#analyzers)
+## Table of Contents
 
-## Organization
+  1. [References](#references)
 
-### Directory Structure
+## References
 
-This structure is an adaptation of [Sass Guidelines: The 7-1 Pattern]. It represents the organization of different types of style sheets, and the order they should follow. Unless specific project requirements dictate otherwise, the directory structure should look _something_ like this:
+  <a name="references--prefer-const"></a><a name="1.1"></a>
+  - [1.1](#references--prefer-const) Use `const` for all of your references; avoid using `var`. 
```

Comment:
I find this structure of numbered bullets kinda hard to read. 😕 

Any reason why we wouldn't use subheadings to more clearly differentiate the rules, which would also give us hash-anchors for free in GitHub? It just feels to me like we're overloading list items a bit.